### DISCUSSION TITLE
Sorting logic by "Action" in Programs list

### DISCUSSION THREAD

**Discussion by rockenren** [tnodir/fort#184] [Q&A]
Fort 3.10.6.

If you have blocked apps by default action "Block, if not allowed" AND have manually blocked apps, then sorting by "Action" column works incorrectly in comp with other columns:

- if sorting by ASC, the order of sorting is: **B**lock (Auto), **A**llow, **B**lock (Manual).

![image](https://github.com/tnodir/fort/assets/11333307/9e2b4b36-b7a8-432f-ba0a-b77377c561b6)

Which type of sorting is used here?

IMO it's kinda confusing, especially if you have many items in list.

**Comment by tnodir** [comment]
Sorting by Action:
- `Alerted` items are always on top
- then `Allowed` / `Blocked` items
- then `Kill Process` items are always on bottom

**Comment by rockenren** [comment]
Got it, thanks.
